tests/conftest.py

```diff
@@ -41,7 +41,9 @@ class ResourceTableIdPos(IntEnum):
     CONDITION = 7
     DOCUMENTREFERENCE = 11
     ENCOUNTER = 20
-    OBSERVATION = 14
+    MEDICATION = 1
+    MEDICATIONREQUEST = 1
+    OBSERVATION = 1
```

Comment:
So, I knocked observation ID to the first col since I was already doing a bunch of changes to get everything unnested correctly. 

Either at the end of the PR review, or as a followon, I am going to move the primary ID of each core table to the first column, and then remove this check. I just didn't want to pollute the test data changes too much while there were other structural changes going on, just to make it more obvious what was actually changing as part of this.